lib/Mojolicious/Command/nopaste/Service/fpaste.pm

```diff
@@ -0,0 +1,35 @@
+package Mojolicious::Command::nopaste::Service::fpaste;
+use Mojo::Base 'Mojolicious::Command::nopaste::Service';
+
+has description => "Post to fpaste.org\n";
+
+sub paste {
+  my $self = shift;
+  my $lang = $self->language || 'text';
```

Comment:
most (all?) of the services default to perl syntax so far. this is of debatable value, but until a change is made, perhaps aim for consistency?